cfme/tests/physical_infrastructure/test_physical_server_inventory_api.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils import testgen
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3)]
+
+pytest_generate_tests = testgen.generate([LenovoProvider], scope="class")
+
+
+@pytest.fixture(scope='class')
+def api(appliance):
+    return appliance.rest_api
+
+
+@pytest.yield_fixture(scope="class")
+def physical_server(provider, appliance):
+    if not provider.exists:
+        provider.create_rest()
+    provider.refresh_provider_relationships
+    provider.validate()
+    physical_server = appliance.rest_api.collections.physical_servers[0]
+    yield physical_server
```

Comment:
OPTIONAL: no need to use ``yield_fixture`` here since no actions follow after ``yield``.